### PR TITLE
Update Asymmetric and Symmetric Encryption pages

### PR DESCRIPTION
**Pull request by YvonneW**
Following pages were added:

- docs/administer/adding-multiple-keys-to-the-primary-keystore.md
- docs/administer/configuring-keystores-in-wso2-products.md
- docs/administer/enabling-java-security-manager.md
- docs/administer/migrating-the-secondary-userstore-password-to-the-internal-keystore.md
- docs/administer/using-symmetric-encryption.md

The following files were updated: 
- docs/administer/creating-new-keystores.md
- docs/administer/managing-keystores-with-the-ui.md
- docs/administer/renewing-a-ca-signed-certificate-in-a-keystore.md
- docs/administer/using-asymmetric-encryption.md 
- mkdocs.yml

Removed 'Enabling Java Security Manager' section from the 'Administrator > Product Administration > Implementing Security in the Identity Server' section.

### CODE REVIEW

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+	      "e": "AQAB",
+	      "use": "sig",
+	      "kid": "NTAxZmMxNDMyZDg3MTU1ZGM0MzEzODJhZWI4NDNlZDU1OGFkNjFiMQ_RS256",
+	      "alg": "RS256",
+	      "n": "luZFdW1ynitztkWLC6xKegbRWxky-5P0p4ShYEOkHs30QI2VCuR6Qo4Bz5rTgLBrky03W1GAVrZxuvKRGj9V9-PmjdGtau4CTXu9pLLcqnruaczoSdvBYA3lS9a7zgFU0-s6kMl2EhB-rk7gXluEep7lIOenzfl2f6IoTKa2fVgVd3YKiSGsyL4tztS70vmmX121qm0sTJdKWP4HxXyqK9neolXI9fYyHOYILVNZ69z_73OOVhkh_mvTmWZLM7GM6sApmyLX6OXUp8z0pkY-vT_9-zRxxQs7GurC4_C1nK3rI_0ySUgGEafO1atNjYmlFN-M3tZX6nEcA6g94IavyQ"
+	    }
+	  ]
+	}
+	```
+
```
Comment (ayshsandu) [inline en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: remove unnecessary lines

File: en/docs/administer/configuring-keystores-in-wso2-products.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        2.  Execute the following command.
+            ```
+            grep -nr ".jks"
+            ``` 
+
+            Note that the configuration files and the keystore files referred to in each file are listed out. Following is an example: 
+
+
+            ```text
+            ./axis2/axis2.xml:260:                <Location>repository/resources/security/wso2carbon.jks</Location>
```
Comment (ayshsandu) [inline en/docs/administer/configuring-keystores-in-wso2-products.md]: These .xml config are no longer valid. Need to update with proper toml based config.

File: en/docs/administer/configuring-keystores-in-wso2-products.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+The `wso2carbon.jks` keystore file, which is shipped with WSO2 Identity Server, is used as the default keystore for all functions. However, in a production environment, it is recommended to create new keystores with new keys and certificates.
+    
+!!! tip "Before you begin"
+    1.  If you want an easy way to locate all the configuration files that have references to keystores, you can use the grep command as follows:
+
+        1.  Navigate to the `<IS_HOME>/repository/conf/` directory on a command prompt.
+
+        2.  Execute the following command.
+            ```
+            grep -nr ".jks"
```
Comment (ayshsandu) [inline en/docs/administer/configuring-keystores-in-wso2-products.md]: This step may not needed in toml based model. Verify the config for new model and update

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md [hunk 4]
```diff
@@ -0,0 +1,139 @@
+# Adding Multiple Keys to the Primary Keystore
+
+WSO2 Identity Server enables adding multiple keys to the primary keystore. Let's explore the following topics to learn more.
+
+!!! tip "Before you begin"
+
+	Sign in to the WSO2 Identity Server Management Console that will be hereafter called Management Console.
+
```
Comment (omindu) [inline en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: Remove this section and link the 'Management Console' phrase to the respective 'Management Console' section.

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+
+## Adding new keys to the super tenant
```
Comment (omindu) [inline en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: Add a brief description about why we should add multiple keys.

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
...
+		```curl tab="Format"
+		keytool -genkey -alias <PUBLIC_CERTIFICATE_ALIAS> -keyalg RSA -keysize 2048 -keystore <KEYSTORE_NAME> -dname "CN=<<Common Name>>,OU=<<Organization Unit>>,O=<<Organization>>,L=<<Locality>>,S=<<StateofProvice Name>>,C=<<Country Name>>"-storepass <KEYSTORE_PASSWORD> -keypass <PRIVATE_KEY_PASSWORD>
+		```
+
+		``` curl tab="Example"
+		keytool -genkey -alias newkey -keyalg RSA -keysize 2048 -keystore wso2carbon.jks -dname "CN=localhost, OU=IT,O=ABC,L=SL,S=WS,C=LK" -storepass wso2carbon -keypass wso2carbon
+		```
+
+	!!! tip
+		If you are planning to delete the newly-added keys in the future, it is recommended to maintain separate keystores for internal and external encryption purposes.
```
Comment (omindu) [inline en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: Let's rephrase this in a way that indicates that you will remove keys where you are periodically doing key rotation.

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md (hunk 5 above)
Comment (omindu) [reply en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: @inthirakumaaran can you provide some content for this section

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+	!!! note
+		Make sure to use the same password for all the keys and add it to the `Password` parameter under the `[keystore.tls]` element in the `deployment.toml` file in the `<IS_HOME>/repository/conf` directory.
```
Comment (omindu) [inline en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: Why?

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md [hunk 8]
```diff
@@ -0,0 +1,139 @@
+# Adding Multiple Keys to the Primary Keystore
+
+WSO2 Identity Server enables adding multiple keys to the primary keystore. Let's explore the following topics to learn more.
```
Comment (omindu) [inline en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: We will have to explain why we should maintain multiple keys and state whether this is only supported for super tenants. @inthirakumaaran please verify.

File: en/docs/administer/using-asymmetric-encryption.md [hunk 9]
```diff
@@ -1,196 +1,95 @@
...
-later case, there can be a chain of CAs involved in signing your public
-key certificate. However, note that both types of public key
-certificates (self-signed or CA-signed) can be effectively used
-depending on the sensitivity of the information that is protected by the
-keys.
-
-In summary, each trust chain entry in a keystore contains the
-following:
+WSO2 Identity Server uses asymmetric encryption by default for the authentication and protection of data. In asymmetric encryption, keystores (with private keys and public key certificates) and
+truststores (with only public key certificates) are created and stored. It is possible to have multiple keystores so that the keys used for different use cases are kept unique. The following topics explain more details on keystores and truststores, and how they are used in WSO2 Identity Server.
```
Comment (omindu) [inline en/docs/administer/using-asymmetric-encryption.md]: `In asymmetric encryption, keystores (with private keys and public key certificates) and
truststores (with only public key certificates) are created and stored.`.

Let's remove this sentence. It doesn't add anything valuable to the information.

Better if we can add a brief intro on what asymmetric encryption is

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md (hunk 1 above)
Comment (YvonneW) [reply en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: Fixed

File: en/docs/administer/configuring-keystores-in-wso2-products.md (hunk 2 above)
Comment (YvonneW) [reply en/docs/administer/configuring-keystores-in-wso2-products.md]: Fixed

File: en/docs/administer/configuring-keystores-in-wso2-products.md (hunk 3 above)
Comment (YvonneW) [reply en/docs/administer/configuring-keystores-in-wso2-products.md]: Fixed

File: en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md (hunk 4 above)
Comment (YvonneW) [reply en/docs/administer/adding-multiple-keys-to-the-primary-keystore.md]: Fixed

File: en/docs/administer/using-asymmetric-encryption.md (hunk 9 above)
Comment (YvonneW) [reply en/docs/administer/using-asymmetric-encryption.md]: Fixed

**Pull request merged**